server/game/basecard.js

```diff
@@ -292,6 +304,22 @@ class BaseCard {
         return !!this.factions[normalizedFaction];
     }
 
+    getFactions() {
+        let factions = [];
+
+        _.each(ValidFactions, faction => {
+            if(this.isFaction(faction)) {
+                factions.push(faction);
```

Comment:
Nitpicky, but this would be more succinctly expressed as:
```javascript
let factions = _.filter(ValidFactions, faction => this.isFaction(faction));
```